homeassistant/components/switch/rpi_rf.py

```diff
@@ -27,8 +27,8 @@
 DEFAULT_SIGNAL_REPETITIONS = 10
 
 SWITCH_SCHEMA = vol.Schema({
-    vol.Required(CONF_CODE_OFF): cv.positive_int,
-    vol.Required(CONF_CODE_ON): cv.positive_int,
+    vol.Required(CONF_CODE_OFF): cv.string,
```

Comment:
Looking into Voluptuous, is there any way to further validate the string to verify that it is composed entirely of comma-separated ints? Seems like this change will lose the ability to ensure no letters made it into the codes and detect that during config validation (prior to run-time). Is there perhaps a way to at least specify ranges of valid characters e.g. `[0-9,]`?

EDIT: Another option that would preserve config validation may be to have the config be a yaml array of codes.